src/command.ts

```diff
@@ -76,7 +76,7 @@ function addCompareSnapshotCommand(screenshotOptions?: ScreenshotOptions): void
         }
       }
 
-      const nameSanitized = name.replace(/[^a-z0-9]/gi, '_').toLowerCase()
+      const nameSanitized = name.replace(/[<>:"/\\|?*]/g, '_')
```

Comment:
You are not scaping space " ". Is that intentional?
I'm wondering how it will eventually work on linux / windows servers.

Reply:
maybe we can also do a test for this use case to be safe

Reply:
In the meantime, I have actually completely removed the custom sanitation from the frontend and only rely on sanitize from the [sanitize-filename](https://www.npmjs.com/package/sanitize-filename) dependency. In the meantime I'll add the space character tests and handle it if needed

Reply:
seems to be working without issues

Reply:
I specifically added a 'file names' tests to run on all platforms, and they all passed